media-sound/vimpc/vimpc-0.09.2.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit autotools
+
+DESCRIPTION="An ncurses based mpd client with vi-like key bindings"
+HOMEPAGE="https://github.com/boysetsfrog/vimpc"
+SRC_URI="https://github.com/boysetsfrog/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="boost taglib"
+
+RDEPEND="dev-libs/libpcre
+	media-libs/libmpdclient
+	boost? ( dev-libs/boost )
```

Comment:
you definitely want to trigger rebuild on boost subslot bump, so `:=` it is

Reply:
Thanks! Fixed

Reply:
The same is in vimpc-0.09.1 ebuild, and it is stable. Is a straight to stable fix needed?

Reply:
no, fix will be stabilising .2 instead.